WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -227,10 +227,11 @@ extension WPRichContentView: WPTextAttachmentManagerDelegate {
 
         attachment.maxSize = CGSize(width: width, height: height)
 
-        if attachment.tagName == "iframe" {
-            embed.loadContentURL(URL(string: attachment.src.stringByDecodingXMLCharacters())!)
+        if let url = URL(string: attachment.src.stringByDecodingXMLCharacters()), attachment.tagName == "iframe" {
```

Comment:
Would it be more performant to check `attachment.tagName == "iframe"` first, before running `URL(string: attachment.src.stringByDecodingXMLCharacters())`?